test/lib/confirm-controller.js

```diff
@@ -1,126 +1,409 @@
 'use strict';
 
+const path = require('path');
 const proxyquire = require('proxyquire');
+const i18nFuture = require('i18n-future');
 
-const Controller = sinon.stub();
-Controller.prototype = {};
-
-const ConfirmController = proxyquire('../../lib/confirm-controller', {
-  './base-controller': Controller
-});
-
-describe('lib/confirm-controller', () => {
-  const req = {
-    params: {},
-    translate: function () {}
-  };
-  const res = {};
-  let controller;
-
-  const modifierSpy = sinon.spy((value) => value * 3);
+describe('Confirm Controller', () => {
+  class StubController {
+    constructor(options) {
+      this.options = options;
+    }
+  }
+  class EmailerStub {}
+  let ConfirmController;
+  let confirmController;
+  let helpersStub;
 
   beforeEach(() => {
-    controller = new ConfirmController({
-      template: 'foo'
-    });
-    controller.getErrors = sinon.stub().returns({foo: true});
-    res.locals = {};
-    res.locals.values = {
-      'field-one': 1,
-      'field-two': 2,
-      'field-three': 3,
-      'field-four': 4
-    };
-
-    controller.options = {
-      steps: {
-        '/one': {
-          fields: ['field-one', 'field-two']
+    StubController.prototype.locals = sinon.stub().returns({});
+    StubController.prototype.get = sinon.stub();
+    EmailerStub.prototype.sendEmails = sinon.stub().returns(new Promise(resolve => resolve()));
+    /* eslint-disable no-underscore-dangle */
+    EmailerStub.prototype._initApp = sinon.stub();
+    EmailerStub.prototype._initEmailer = sinon.stub();
+    EmailerStub.prototype._includeDate = sinon.stub();
+    /* eslint-enable no-underscore-dangle */
+  });
+
+  describe('with stubbed helpers', () => {
+    beforeEach(() => {
+      helpersStub = {
+        getStepFromFieldName: sinon.stub(),
+        isEmptyValue: sinon.stub(),
+        hasOptions: sinon.stub(),
+        getValue: sinon.stub(),
+        conditionalTranslate: sinon.stub(),
+        getTranslation: sinon.stub()
+      };
+      ConfirmController = proxyquire('../../lib/confirm-controller', {
+        './base-controller': StubController,
+        './util/helpers': helpersStub,
+        'hof-emailer': EmailerStub
+      });
+      const steps = {
```

Comment:
yeah I guess we could split it out, where do we draw the line though - do we add the field config to a separate file  (7 lines). The idea of fixtures were more real use case like scenarios, whereas these 'fixtures' are testing a simple use case